dali/python/nvidia/dali/_utils/eager_utils.py

```diff
@@ -209,6 +213,9 @@ def _slice_tensorlist(data, size):
     return type(data)(list(data)[:size], layout=data.layout())
 
 
+# TODO(klecki): Expose `nvidia.dali.math` in eager mode
+
+
```

Comment:
I have reservations with leaving a TODO in code. Does that mean anything? ;)

Reply:
I hope. This is experimental module which we currently deprioritized, so maybe if we pick it up again we will clear this one.